controller/adoptopenebs/jiva.go

```diff
@@ -0,0 +1,20 @@
+package adoptopenebs
+
+import "k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+// formJivaConfig forms the desired OpenEBS CR config for Jiva.
+func (p *Planner) formJivaConfig() error {
+	jivaConfig := &unstructured.Unstructured{
+		Object: make(map[string]interface{}, 0),
+	}
+	if p.JivaConfig != nil {
+		jivaConfig = p.JivaConfig
+	}
+	if p.JivaCtrlImageTag != p.OpenEBSVersion {
```

Comment:
why is adopt updating the image?

Reply:
It is not updating the image, it's checking if some different image tag is getting used for any component, if it is the case, it will update the OpenEBS CR with that particular image tag for the respective component.